.github/workflows/release-state-manager.yml

```
@@ -0,0 +1,113 @@
+name: Publish State Manager image to GHCR and Deploy to K8s
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)